copier/main.py

```diff
@@ -1,4 +1,5 @@
 """Main functions and classes, used to generate or update projects."""
+
```

Comment:
Those empty lines were introduced by Ruff v0.3.0 (IIRC). Would you mind reverting those (and the other occurences), too? They aren't related to this PR.